src/styles/blocks/main.scss

```
@@ -0,0 +1,17 @@
+.main {
```

Add a background-color to `.main` to create the fixed background effect.
```suggestion
.main {
  background-color: $main-text-color-black;
```